### ISSUE TITLE
[Question] Does `checkm lineage_wf` use CPR_43 marker set for Patescibacteria? 

### ISSUE THREAD

**Issue by jolespin** [Ecogenomics/CheckM#304]
I had a question regarding your recent update:  https://github.com/Ecogenomics/CheckM/wiki/Workflows#using-cpr-marker-set

If one is running `checkm lineage_wf` and a Patescibacteria is found, is the marker set automatically [CPR_43](https://github.com/Ecogenomics/CheckM/blob/master/custom_marker_sets/cpr_43_markers.hmm) for this?  

Also, I think https://github.com/Ecogenomics/CheckM/tree/master/custom_marker_sets/cpr_43_markers.xlxs is mispelled and is supposed to be cpr_43_markers.xlsx.  I downloaded it and confirmed it was an excel doc.

**Comment by dparks1134** [comment]
Unfortunately no. If your MAGs are Patescibacteria you need to run the CPR43 marker set on them explicitly. We are working on a new version of CheckM that will address this short coming.

**Comment by jolespin** [comment]
Awesome, that's what I was thinking but I wanted to double check before I started filtering out my bins.

**Comment by jolespin** [comment]
Do you know when this might be integrated in? If not, I can maybe trying to hack something together as a prototype.  

Do you have any in suggestions on what code to edit? 

I'm building some pipelines that use this and filter out low quality samples and don't want to miss out on CPR?

**Comment by dparks1134** [comment]
Hi. We are working on CheckM version 2 so there are no plans to incorporate the CPR marker set into CheckM v1. Adding in this marker set would be complicated so you are probably better off trying to identify any CPR MAGs first and then running CheckM with the CPR marker set to assess their quality.

**Comment by jolespin** [comment]
Good call.  I took a look at it and I don't know the mechanics of CheckM well enough to do it justice.  I made a work around for one of my scripts.  I ran lineage_wf on a bunch of bona fide CPR and almost ever single one came back as basal bacteria lineage for the marker set.  I just set an argument to include these when running my completion/contamination filters so I can adjust post hoc. 

Looking forward to CheckM version 2. Such an iconic piece of software in metagenomics. 

If you ever need a 3rd party tester, I have a lot of datasets at JCVI I can run it on.
